wcfapp/WcfServiceWebApp/Web.Debug.config

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+
```

Comment:
Do we want to leave these unused template items in the project? They aren't needed for this sample.

Reply:
Well, it will remove the `debug=true` from `compilation` node for release build, so not completely unused. I would leave them there as they are generated by VS template. Strictly speaking, we don't need `AssemblyInfo.cs` either. 

The main reason I don't want to do over-cleanup (including those default comments) is so that users can see most things are just like what they are familiar with - generated from VS default template - they don't have to spend the time to understand every single change to see if they are necessary for the sample or just a cleanup.